diff -urN aaa/gdl-0.4.0/symbol-browser-control/factory.c gdl-0.4.0/symbol-browser-control/factory.c
--- aaa/gdl-0.4.0/symbol-browser-control/factory.c	2003-01-12 05:34:18.000000000 +0100
+++ gdl-0.4.0/symbol-browser-control/factory.c	2003-08-12 00:39:15.000000000 +0200
@@ -22,6 +22,7 @@
 #include "config.h"
 #endif
 
+#include <locale.h>
 #include <bonobo.h>
 #include <bonobo-activation/bonobo-activation.h>
 #include <gdl/gdl.h>
diff -urN aaa/gdl-0.4.0/symbol-browser-control/test-symbol-browser.c gdl-0.4.0/symbol-browser-control/test-symbol-browser.c
--- aaa/gdl-0.4.0/symbol-browser-control/test-symbol-browser.c	2003-01-12 05:34:18.000000000 +0100
+++ gdl-0.4.0/symbol-browser-control/test-symbol-browser.c	2003-08-12 00:43:29.000000000 +0200
@@ -1,4 +1,5 @@
 #include <config.h>
+#include <locale.h>
 #include <bonobo.h>
 #include <gnome.h>
 #include <bonobo-activation/bonobo-activation.h>
