--- gdl-3.40.0/gdl/gdl-dock-item.c.orig	2018-10-22 21:36:04.000000000 +0200
+++ gdl-3.40.0/gdl/gdl-dock-item.c	2025-08-06 12:51:12.492830761 +0200
@@ -355,7 +355,7 @@
     container_class->set_focus_child = gdl_dock_item_set_focus_child;
     gtk_container_class_handle_border_width (container_class);
 
-    gdl_dock_object_class_set_is_compound (object_class, FALSE);
+    gdl_dock_object_class_set_is_compound (dock_object_class, FALSE);
     dock_object_class->dock_request = gdl_dock_item_dock_request;
     dock_object_class->dock = gdl_dock_item_dock;
     dock_object_class->present = gdl_dock_item_present;
@@ -744,7 +744,7 @@
         case PROP_RESIZE:
             item->priv->resize = g_value_get_boolean (value);
             {
-                GObject * parent = gtk_widget_get_parent (GTK_WIDGET (item));
+                GObject * parent = G_OBJECT(gtk_widget_get_parent (GTK_WIDGET (item)));
                 //if we docked update "resize" child_property of our parent
                 if(parent)
                 {
--- gdl-3.40.0/gdl/gdl-dock-layout.c.orig	2018-10-22 21:36:04.000000000 +0200
+++ gdl-3.40.0/gdl/gdl-dock-layout.c	2025-08-06 12:52:57.783303292 +0200
@@ -620,7 +620,7 @@
         if (GDL_IS_DOCK_OBJECT (master)) {
             master = gdl_dock_object_get_master (GDL_DOCK_OBJECT (master));
         }
-        layout->priv->master = g_object_ref (master);
+        layout->priv->master = g_object_ref (GDL_DOCK_MASTER(master));
         layout->priv->layout_changed_id =
             g_signal_connect (layout->priv->master, "layout-changed",
                               (GCallback) gdl_dock_layout_layout_changed_cb,
--- gdl-3.40.0/gdl/gdl-dock-bar.c.orig	2018-10-22 21:36:04.000000000 +0200
+++ gdl-3.40.0/gdl/gdl-dock-bar.c	2025-08-06 12:54:32.342436204 +0200
@@ -425,7 +425,7 @@
         if (GDL_IS_DOCK_OBJECT (master)) {
             master = gdl_dock_object_get_master (GDL_DOCK_OBJECT (master));
         }
-        dockbar->priv->master = g_object_ref (master);
+        dockbar->priv->master = g_object_ref (GDL_DOCK_MASTER(master));
         dockbar->priv->layout_changed_id =
             g_signal_connect (dockbar->priv->master, "layout-changed",
                               (GCallback) gdl_dock_bar_layout_changed_cb,
